packages/safe-ds-lang/src/language/generation/safe-ds-python-generator.ts

```diff
@@ -368,7 +375,13 @@ export class SafeDsPythonGenerator {
         frame: GenerationInfoFrame,
         generateLambda: boolean = false,
     ): CompositeGeneratorNode {
+        let targetPlaceholder = frame.targetPlaceholder
+            ? getPlaceholderByName(block, frame.targetPlaceholder)
+            : undefined;
```

Comment:
Allow the second parameter of `getPlaceholderByName` to be `undefined` and return `undefined` in that case. This makes it easier for the caller, since they can avoid conditionals. We've frequently used this pattern for other methods.